<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Mapper</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="outputTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>123</height>
     </rect>
    </property>
    <property name="title">
     <string>Remote Mappings</string>
    </property>
    <widget class="QPushButton" name="showKeysBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>set hotkeys</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendKeysBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>send keys</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>281</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
